functions/Restore-DbaDatabase.ps1

```diff
@@ -557,7 +557,7 @@ function Restore-DbaDatabase {
 		
 		
         if ($AllFilteredFiles.count -gt 1 -and $DatabaseName -ne '') {
-            Write-Warning "$FunctionName -  DatabaseName parameter and multiple database restores is not compatible "
+            Write-Message -Level Warning -Message "$FunctionName -  DatabaseName parameter and multiple database restores is not compatible " 
             break
```

Comment:
Use Stop-Function instead. Including the function name in messages is no longer needed.